package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "oak.js",
-  "version": "1.9.0-rc.4",
+  "version": "2.0.0-rc.4",
```

Comment:
This version needs to be reset to "2.0.0-rc.1", meaning release candidate 1 for 2.0.0